lib/dal/src/prop.rs

```diff
@@ -1129,31 +1165,6 @@ impl Prop {
         Ok(props)
     }
 
-    pub async fn modify<L>(self, ctx: &DalContext, lambda: L) -> PropResult<Self>
```

Comment:
Is this unused? seems likely

Reply:
Yep, it all compiles after removing, so we decided to remove rather than fix it to use PropContentV2.